python/tvm/relay/op/_tensor.py

```diff
@@ -27,6 +27,7 @@
 
 
 register_broadcast_schedule("log")
+register_broadcast_schedule("tan")
```

Comment:
Add shape function for tan as well?

Reply:
Hi ! What do you mean by adding shape function for tan ?

Reply:
https://github.com/apache/incubator-tvm/blob/b6893017d964f531e7f1600faaec4e3817fe89e2/python/tvm/relay/op/_tensor.py#L185-L217